apps/web/app/api/v1/client/[environmentId]/app/sync/lib/utils.ts

```diff
@@ -0,0 +1,81 @@
+import { parseRecallInfo } from "@formbricks/lib/utils/recall";
+import { TLegacySurvey } from "@formbricks/types/LegacySurvey";
+import { TAttributes } from "@formbricks/types/attributes";
+import { TSurvey } from "@formbricks/types/surveys";
+
+export const checkForAttributeRecall = (survey: TSurvey, attributes: TAttributes): TSurvey => {
```

Comment:
I would assume that a `check...' method would return true or false, but this function modifies the survey and returns the modified one. So maybe something like `replaceAttributeRecalls()` would be more appropriate?
(still not sold on it, but maybe in that direction)